.github/workflows/server-lint.yml

```diff
@@ -0,0 +1,32 @@
+name: Server Lint & Code Quality
```

Comment:
file is wrongly placed. was renamed in a recent merge to "server.yml". Please check your files.